updating the documentation to support accio (#145)
smartdevicelink/sdl_ios_guides: docs/Getting Started/Installation/index.md

In order to build your app on a SmartDeviceLink (SDL) Core, the SDL software development kit (SDK) must be installed in your app. The following steps will guide you through adding the SDL SDK to your workspace and configuring the environment.

## Install SDL SDK
There are four different ways to install the SDL SDK in your project: Accio, CocoaPods, Carthage, or manually.

### Accio

> You can install this library using [Accio/SwiftPM](https://github.com/JamitLabs/Accio) documentation page.  Please follow the steps to install and initialization Accio into a current or new application. 

Reviewer: ```suggestion
You can install this library using [Accio](https://github.com/JamitLabs/Accio), which is based on SwiftPM syntax. Please follow the steps on the Accio README linked above to initialize Accio into your application.
```

> Once installed and initialized into your xcode project the root directory should contain a Package.swift file.

Reviewer: ```suggestion
Once installed and initialized into your Xcode project, the root directory should contain a Package.swift file.
```

> 1. Open the Package.swift file 

Reviewer: ```suggestion
1. Open the Package.swift file.
```

> 2. Add the following line the dependencies array of you Package. We suggest always using the latest release of the SDL library. 

Reviewer: ```suggestion
2. Add the following line to the dependencies array of your package file. We suggest always using the latest release of the SDL library. 
```

        .package(url: "https://github.com/smartdevicelink/sdl_ios.git", .upToNextMajor(from: "6.2.0"),
        

> !!! NOTE

Reviewer: Try indenting the NOTE to keep the numbering working.

> Please see [Mainifest format](https://github.com/apple/swift-package-manager/blob/master/Documentation/PackageDescriptionV4.md) to specify dependencies to a specific branch / version of SDL.

Reviewer: ```suggestion
Please see [package manifest format](https://github.com/apple/swift-package-manager/blob/master/Documentation/PackageDescriptionV4.md) to specify dependencies to a specific branch / version of SDL.
```

!!!
            

> 3. Add `"SmartDeviceLink"` or `"SmartDeviceLinkSwift"` to your dependencies array in your target. Use `"SmartDeviceLink"`for Objective-C applications and `"SmartDeviceLinkSwift"` for Swift applications.

Reviewer: ```suggestion
3. Add `"SmartDeviceLink"` or `"SmartDeviceLinkSwift"` to the dependencies array in your target. Use `"SmartDeviceLink"`for Objective-C applications and `"SmartDeviceLinkSwift"` for Swift applications.
```

In order to build your app on a SmartDeviceLink (SDL) Core, the SDL software development kit (SDK) must be installed in your app. The following steps will guide you through adding the SDL SDK to your workspace and configuring the environment.

## Install SDL SDK
There are four different ways to install the SDL SDK in your project: Accio, CocoaPods, Carthage, or manually.

### Accio

> 

Reviewer: Remove this extra line

You can install this library using [Accio/SwiftPM](https://github.com/JamitLabs/Accio) documentation page.  Please follow the steps to install and initialization Accio into a current or new application. 

> Once installed and initialized into your xcode project the root directory should contain a Package.swift file

Reviewer: Add a period at the end

1. Open the Package.swift file 

> 2. Add the follwing line the dependencies array of you Package. We suggest always using the latest release. 

Reviewer: `follwing` -> `following`

`you` -> `your`

`latest release` -> `latest release of the SDL library`

Author: changed

>     Please see [Mainifest format](https://github.com/apple/swift-package-manager/blob/master/Documentation/PackageDescriptionV4.md) to specify dependencies to a specific branch / version of SDL.

Reviewer: Move below the code line and wrap in

!!!NOTE
"Text"
!!!

>         .package(url: "https://github.com/smartdevicelink/sdl_ios.git", .from: "6.2.3"),

Reviewer: Change `.from` to `.upToNextMajor(from: "6.2.0")`

Author: changed

>         .package(url: "https://github.com/smartdevicelink/sdl_ios.git", .from: "6.2.3"),

Reviewer: Wrap in swift code block tags

Author: I'm unsure what you mean by this. The code shows up formatted on the documentation page

Reviewer: We don't format code by using indentation. We format code using backticks. See other guides. This is technically swift code, and should be formatted as such using "```swift"

> 3. Add the following to the dependencies array of the target.

Reviewer: Change to:

`Add "SmartDeviceLink" or "SmartDeviceLinkSwift" to your dependencies array in your target. Use "SmartDeviceLink" for Objective-C applications and "SmartDeviceLinkSwift" for Swift applications.`

        "SmartDeviceLink"
            
!!! NOTE

> If creating a swift application please `"SmartDeviceLinkSwift"` instead.

Reviewer: Remove this note based on the above change

!!!

> 1. Install SDL SDK for iOS: 

Reviewer: "Install the SDK by running `accio install` in the root folder of your project in Terminal.

> 1. Install SDL SDK for iOS: 

Reviewer: This should be a 4

>         accio install

Reviewer: Remove
